python/cuml/cuml/dask/naive_bayes/naive_bayes.py

```diff
@@ -76,7 +76,7 @@ class MultinomialNB(BaseEstimator, DelayedPredictionMixin):
 
         >>> # Compute accuracy on training set
         >>> model.score(X, y)
-        array(0.924...)
+        array(0.9...)
```

Comment:
Fine by me, though for things like this a `# doctest: +SKIP` would be preferable IMO (or rewriting it to not `repr` a score at all).

Reply:
Hadn't thought of skipping, but I like it. I assume we run the doctests to make sure they work and aren't horribly broken, not as a way to ensure the estimator actually works/doesn't contain algorithmic bugs.